docs/data/data-grid/filtering-recipes/QuickFilterOutsideOfGrid.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import * as ReactDOM from 'react-dom';
+import { Portal } from '@mui/base/Portal';
```

Comment:
Would `export default` usage be more appropriate as the path being used is an absolute one?
```suggestion
import Portal from '@mui/base/Portal';
```

Reply:
This recently changed https://github.com/mui/material-ui/releases/tag/v5.14.4. As I understand it, the idea is to have this for all @mui packages, as each gets a new major. cc @michaldudak